addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -287,6 +287,12 @@ def _get_store_partner_fields(self, fields):
     # RTC (voice/video)
     # --------------------------------------------------------------------------
 
+    def _should_invite_members_to_join_call(self):
```

Comment:
It feels like a method that should be on channel and not on members (only using `self.channel_id` in particular).